benchmarks/graphics_pipeline/main.cpp

```diff
@@ -783,6 +850,60 @@ void ProjApp::Setup()
     }
 }
 
+void ProjApp::KeyDown(ppx::KeyCode key)
+{
+    mPressedKeys.insert(key);
+}
+
+void ProjApp::KeyUp(ppx::KeyCode key)
+{
+    mPressedKeys.erase(key);
+}
+
+void Person::Move(MovementDirection dir, float distance)
+{
+    if (dir == MovementDirection::FORWARD) {
+        location += float3(0, 0, distance);
```

Comment:
This camera will be controlled by a human being, meaning we want some movement regularity, even with a very high/very low framerate. To handle that, you need to multiply each movement vector by deltatime.
(https://medium.com/star-gazers/understanding-time-deltatime-6528a8c2b5c8)

Reply:
Thank you for the clarification, I added it.